plugin/hosts/hostsfile.go

```diff
@@ -135,6 +137,8 @@ func (h *Hostsfile) readHosts() {
 	h.mtime = stat.ModTime()
 	h.size = stat.Size()
 
+	hostsEntries.WithLabelValues().Set(float64(h.inline.Len() + h.hmap.Len()))
+	hostsReloadTime.SetToCurrentTime()
```

Comment:
should we use `ModTime()` here?

Reply:
Thanks for your suggestion. I have changed to use `stat.ModTime()`. It is more appropriate